docs/README.md

```diff
@@ -17,6 +17,7 @@
   * [Haskell (haskell)](./docs/Options.md#haskell-haskell)
   * [Pyenv (pyenv)](./docs/Options.md#pyenv-pyenv)
   * [Go (golang)](./docs/Options.md#go-golang)
+  * [Kubectl (kubectl)](./docs/Options.md#kubectl-context-kubecontext)
```

Comment:
One last nitpick:
I would change the text of the link to `Kubectl context \(`kubecontext`\)` 👍

Reply:
:+1: